src/quarterback/api/methods.ts

```diff
@@ -9,11 +9,6 @@
  *
  * All portions of the code written by Atypon Systems LLC are Copyright (c) 2022 Atypon Systems LLC. All Rights Reserved.
  */
-import { getVersion } from '@manuscripts/transform'
-import {
-  EventSourceMessage,
-  fetchEventSource,
-} from '@microsoft/fetch-event-source'
```

Comment:
Are we going to use SSE for something else? Steps listening was the only usage for that as far as I know.